### ISSUE TITLE
How to export the insightface mxnet model to onnx model which is same as the onnx model given to download.

### ISSUE THREAD

**Issue by flazerain** [onnx/models#240]
I exported the insightface mxnet model with mxnet 1.5 and onnx 1.3 and I cannot build the tensor RT engine with tensorRT 6, the error is:
[TensorRT] ERROR: (Unnamed Layer* 7) [Parametric ReLU]: slope tensor must be unidirectional broadcastable to input tensor
[TensorRT] ERROR: Network must have at least one output

and I found the onnx net structure is different from the onnx model given to download.

How to export the arcface mxnet model that can be converted to tensorRT engine?

**Comment by sky186** [comment]
@flazerain 
Hi you can convert arcface model to onnx right ?, I convert .,but prelu layer out is error

**Comment by flazerain** [comment]
> @flazerain
> Hi you can convert arcface model to onnx right ?, I convert .,but prelu layer out is error

yes,but it is different from the model zoo 's model(it has reshape before prelu)

**Comment by HoangTienDuc** [comment]
hi @sky186  @flazerain 
have you fixed problem above?
 i also try to convert arcface LResNet100E-IR [mxnet](https://github.com/deepinsight/insightface/wiki/Model-Zoo) to onnx by using [convert_onnx.py](https://github.com/deepinsight/insightface/tree/master/deploy). Then, it seem that, i got error with PRelu  when i deploy my model.
```
onnx runtime error 1: Non-zero status code returned while running PRelu node. Name:'relu0' Status Message: relu0: right operand cannot broadcast on dim 0 LeftShape: {1,64,112,112}, RightShape: {64}
```
Can you guide me how to fix it? 
Thank all off u.

**Comment by theVmagnificient** [comment]
Have anyone resolved the issue?

**Comment by bruceche11** [comment]
I convert the mxnet model (model-r100-ii) to onnx  by using the  insightface/deploy/convert_onnx.py  and get r100.onnx
this step is good.
however,when  do   ../onnx-tensorrt/build/onnx2trt r100.onnx -o r100_engine.trt, thing got wrong:
![image](https://user-images.githubusercontent.com/16239670/106600910-fc28d900-6595-11eb-8225-92d7561b07fc.png)
I do this in nvidia-docker

**Comment by cyrusbehr** [comment]
Try they export scripts here: https://github.com/deepinsight/insightface/issues/1350
